Tasks/PublishTestResultsV2/publishtestresults.ts

```diff
@@ -86,7 +86,11 @@ async function run() {
         if (isNullOrWhitespace(searchFolder)) {
             searchFolder = tl.getVariable('System.DefaultWorkingDirectory');
         }
-
+            tl.debug(tl.getVariable('System.DefaultWorkingDirectory'));
+        if(tl.getVariable('System.DefaultWorkingDirectory')!=null && (!path.isAbsolute(searchFolder)))
```

Comment:
and in JavaScript/TypeScript you can directly check if `if(tl.getVariable('System.DefaultWorkingDirectory') && (!path.isAbsolute(searchFolder)))`

Reply:
and it's a good way in javascript it covers lot of things.